src/components/content/content-theme.scss

```diff
@@ -1,6 +1,4 @@
 md-content.md-THEME_NAME-theme {
   color: '{{foreground-1}}';
-  background-color: '{{background-default}}';
+  background-color: '{{background-color}}';
```

Comment:
this is simply adding back functionality that was introduced in commit 0df4b16eca2c8af9805615da646d38db04a33d7f on March 18, 2015. The md-content component has always allowed for md-hue-x since v0.9.0.